### PR TITLE
Base of page done

### PR DESCRIPTION
**Pull request by TurnbulA**
Set up burger menu location for functionality to be added 
Welcome section completed
Projects area ready for button addition

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Document</title>
```
Comment (mali891) [inline index.html]: What's the name of the project?

File: style.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  font-family: "Roboto", sans-serif;
+}
+body {
+  background-color: #252a41;
+  font-size: 3rem;
+}
+* {
+  box-sizing: border-box;
+}
+.u-text--h {
```
Comment (mali891) [inline style.css]: What does this modifier mean?

File: style.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  color: #fbfcfc;
+}
+.u-text--p {
```
Comment (mali891) [inline style.css]: What does this modifier mean?

File: style.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  margin-left: 2rem;
+  font-size: 1.6rem;
+  width: 100%;
+  height: 100%;
+  border: none;
+  background-color: transparent;
+  color: #bec4dc;
+  outline: none;
+}
+form {
```
Comment (mali891) [inline style.css]: `.c-form`

File: Resources/Burger-icon.svg [hunk 5]
```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="74" height="74" viewBox="0 0 74 74">
+  <g id="Burger-icon" transform="translate(-190 -596)">
```
Comment (mali891) [inline Resources/Burger-icon.svg]: Does this need an id?

File: Resources/Burger-icon.svg [hunk 6]
```diff
@@ -0,0 +1,8 @@
...
+    <rect id="Rectangle_1" data-name="Rectangle 1" width="74" height="74" rx="26" transform="translate(190 596)" fill="#3b3f54"/>
```
Comment (mali891) [inline Resources/Burger-icon.svg]: Do any of these need ids?

File: Resources/Search.svg [hunk 7]
```diff
@@ -0,0 +1,5 @@
+<svg id="search_1_" data-name="search (1)" xmlns="http://www.w3.org/2000/svg" width="39.64" height="39.64" viewBox="0 0 39.64 39.64">
```
Comment (mali891) [inline Resources/Search.svg]: Do any of these need ids?

File: index.html [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            width="74"
+            height="74"
+            viewBox="0 0 74 74"
+            class="burger-icon"
+          >
+            <g id="Burger-icon" transform="translate(-190 -596)">
+              <rect
+                id="Rectangle_1"
```
Comment (mali891) [inline index.html]: Do any of these need ids?

File: index.html [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+      <div class="c-main">
+        <section class="c-header">
+          <h1 class="c-header__text u-text--h">Hello There</h1>
+          <p class="c-header__text u-text--p">
+            Welcome back to the workspace, we missed You!
+          </p>
+          <div class="c-search-bar">
+            <svg
+              id="search_1_"
+              data-name="search (1)"
```
Comment (mali891) [inline index.html]: What's this for?

File: index.html [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+              xmlns="http://www.w3.org/2000/svg"
+              width="39.64"
+              height="39.64"
+              viewBox="0 0 39.64 39.64"
+              class="search-icon"
+            >
+              <g id="Group_8" data-name="Group 8" transform="translate(0 0)">
+                <path
+                  id="Path_1"
```
Comment (mali891) [inline index.html]: Do any of these need ids?

File: index.html (hunk 9 above)
Comment (TurnbulA) [reply index.html]: Search button on search bar 
ID has been removed

File: style.css [hunk 11]
```diff
@@ -1,6 +1,13 @@
+@font-face {
+  font-family: "avnir";
+  src: url(/Resources/avnir_sv/font/AvenirLTStd-Book) format("avnir-book");
```
Comment (mali891) [inline style.css]: Avenir

**Pull request merged**